ui/components/app/modals/export-private-key-modal/password-input.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  BLOCK_SIZES,
+  FLEX_DIRECTION,
+  DISPLAY,
+  AlignItems,
+  Color,
+  TextVariant,
+} from '../../../../helpers/constants/design-system';
+import Box from '../../../ui/box';
+import { useI18nContext } from '../../../../hooks/useI18nContext';
+import { TEXT_FIELD_TYPES, Text, TextField } from '../../../component-library';
+
+const PasswordInput = ({ setPassword }) => {
+  const t = useI18nContext();
+
+  return (
+    <Box
+      width={BLOCK_SIZES.FULL}
+      flexDirection={FLEX_DIRECTION.COLUMN}
+      display={DISPLAY.FLEX}
+      alignItems={AlignItems.flexStart}
+      paddingLeft={5}
+      paddingRight={5}
+    >
+      <Text
+        as="span"
+        color={Color.textDefault}
+        marginBottom={2}
+        variant={TextVariant.bodySm}
+      >
+        {t('typePassword')}
+      </Text>
```

Comment:
There is a dedicated `Label` component that you could use here https://metamask.github.io/metamask-storybook/?path=/docs/components-componentlibrary-label--default-story

Reply:
thanks, i'll use that :+1: